dowhy/causal_refuters/dummy_outcome_refuter.py

```diff
@@ -188,6 +246,8 @@ def refute_estimate(self):
                     # If the number of data points is too few, run the default transformation: [("zero",""),("noise", {'std_dev':1} )]
                     if X_train.shape[0] <= self._min_data_point_threshold:
                         transformation_list = DummyOutcomeRefuter.DEFAULT_TRANSFORMATION
+                        self.logger.warning("Using Default as the number of data points in X_train:{} is less than threshold:{}".format(X_train.shape[0], self._min_data_point_threshold))
+                        self.logger.warning("Usering the transformation list:{}".format(transformation_list))
```

Comment:
The warning can be clearer: What is Default? I think that you mean to say that we only use default transformations, but then line 249 does not make sense. How about "The number of data points {} in the train set for transformations is less than the threshold {}. Therefore, defaulting to using a minimal set of transformations: {}"
Usering-typo

Reply:
Based on the comment above, should also include the category name here.